AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/RemoteSyncEngine.swift

```diff
@@ -228,18 +236,100 @@ class RemoteSyncEngine: RemoteSyncEngineBehavior {
         log.debug(#function)
         let mutationEventClearState = MutationEventClearState(storageAdapter: storageAdapter)
         mutationEventClearState.clearStateOutgoingMutations {
-            if let api = self.api {
-                self.remoteSyncTopicPublisher.send(.clearedStateOutgoingMutations)
-                self.stateMachine.notify(action: .clearedStateOutgoingMutations(api, storageAdapter))
+            self.remoteSyncTopicPublisher.send(.clearedStateOutgoingMutations)
+            self.stateMachine.notify(action: .clearedStateOutgoingMutations)
+        }
+    }
+
+    private func checkIfAuthenticationRequired() {
+        log.debug(#function)
```

Comment:
what will this print? We could have a more informative message here, so it's easier to search in the logs

Reply:
i believe `checkIfAuthenticationRequired()`. added more to all
```
log.debug("\(#function) <message>")
```